qiskit/utils/run_circuits.py

```diff
@@ -224,7 +224,7 @@ def _safe_get_job_status(job: BaseJob, job_id: str, max_job_retries: int) -> Job
             ) from ex
     else:
         raise QiskitError(
-            "job id {}, failed to get the status. Please re-submit job set".format(job_id)
+            "Max retry limit reached. Failed to get status for job with id {}".format(job_id)
```

Comment:
I will note that nowadays we try to use Python f-strings as it makes things a bit more readable in this case it would be 
`f"Max retry limit reached. Failed to get status for job with id {job_id}"` such that the explicit format call is not needed and the variables can be put directly into the placeholders. I only commented on this one but the other places where format is used would be preferably done too using f-string